pysat/instruments/pysat_testing_xarray.py

```diff
@@ -26,7 +33,24 @@
 
 
 # Init method
-init = mm_test.init
+def init(self, test_init_kwarg=None):
+    """Initialize the test instrument.
+
+    Parameters
+    ----------
+    self : pysat.Instrument
+        This object
+    test_init_kwarg : any or NoneType
+        Testing keyword (default=None)
+
+    """
+
+    warnings.warn(" ".join(["The instrument module `pysat_testing_xarray` has",
+                            "been deprecated and will be removed in 3.2.0+."]),
```

Comment:
Recommend pointing user to "pysat_testing2d_xarray" to remove warning.  Also, I think the name "pysat_testing2d_xarray" is the worst name to keep.  Why not deprecate that one and keep this one, swapping any changed data?

Reply:
I agree this should be renamed.  These predate the usage of non-orbital instruments, and could be better clarified.  I've got some suggestions in #908.  Thoughts would be appreciated.